base/strings/basic.jl

```diff
@@ -355,10 +355,14 @@ function nextind(s::AbstractString, i::Integer, nchar::Integer)
         else
             j > e && return j+1
             j == e && return next(s,e)[2]
```

Comment:
And isn't this one redundant, since the loop will be a no-op anyway?

Reply:
Since you've removed the call to `next` below I guess this one needs to stay.

Reply:
exactly